src/build/streams.ts

```diff
@@ -17,7 +17,9 @@ import * as stream from 'stream';
 
 import File = require('vinyl');
 
-(Symbol as any).asyncIterator = Symbol.asyncIterator || Symbol('asyncIterator');
+if (typeof (Symbol as any).asyncIterator  === 'undefined') {
```

Comment:
Prefer:

```typescript
if ((Symbol as any).asyncIterator !== 'undefined') {
  (Symbol as any).asyncIterator = Symbol('asyncIterator');
}
```

`undefined` isn't assignable in modern JS environments, doing the typeof trick is overly paranoid.